mne/epochs.py

```diff
@@ -2600,6 +2768,12 @@ def __init__(self, raw, events, event_id=None, tmin=-0.2, tmax=0.5,
         proj = proj or raw.proj
 
         self.reject_by_annotation = reject_by_annotation
+
+        # store certain raw properties for dealing with annotations
+        self._first_samp = raw.first_samp.copy()
+        self._raw_sfreq = np.rint(raw.info['sfreq'])
+        self._first_time = raw._first_time.copy()
+
```

Comment:
two of these are cruft now; also the `np.rint` was on the wrong one (should go only on samples, non-integer sfreqs are valid)

```suggestion
        # keep track of original sfreq (needed for annotations)
        self._raw_sfreq = raw.info['sfreq']

```

Reply:
We need to keep at least _first_samp in case the Raw was shifted. This is only for the case that `orig_time` and `_first_samp` is set in Annotations and Raw respectively.